tools/pylib/boutdata/data.py

```diff
@@ -235,7 +235,9 @@ class BoutOutputs(object):
     -------
     prefix - sets the prefix for data files (default "BOUT.dmp")
 
-    caching - switches on caching of data, so it is only read into memory when first accessed (defaulte False)
+    caching - switches on caching of data, so it is only read into memory when first accessed (default False)
+              If caching is set to a number, it gives the maximum size of the cache in GB, after which entries will be discarded in first-in-first-out order to prevent the cache getting too big.
+              If the variable being returned is bigger than the maximum cache size, then the variable will be returned without being added to the cache, and the rest of the cache will be left.
```

Comment:
Could you fix that to be not longer than 80 chars?
In emacs e.g. you can enable line wrapping with `M-x auto-fill-mode`